Engine/Client/Src/GraphicalLoad/GraphicalLoad.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+** EPITECH PROJECT, 2024
+** R-Type Client
+** File description:
+** GraphicalLoad
+*/
+
+#include <memory>
+#include <string>
+#include <filesystem>
+
+#include "IGraphic.hpp"
+#include "DLLoader.hpp"
+#include "GraphicalLoad.hpp"
+
+#define PATH_GRAPHIC_LIB "Engine/Client/Lib/Graphic/"
+
+static std::string getPathGraphicalLib()
+{
+    std::string path = PATH_GRAPHIC_LIB;
+
+    for (const auto& file : std::filesystem::directory_iterator(path)) {
+        std::string prefix = path + "libgraphic_";
+        std::string suffix = ".so";
```

Comment:
You should you a define because .so files doesn't exist on Windows, check in the ASceneManager file or in the DLLoader one👍